action.yml

```diff
@@ -4,6 +4,9 @@ branding:
   colour: green
 description: 'Deploy JavaScript applications to Shopify Oxygen directly from GitHub'
 inputs:
+  cloudsmith_token:
```

Comment:
Now that the cli is on npm we can 🔥 this. Tempted to say we could actually merge this now and then tweak later if needed? What do you think?

Reply:
I think we can go ahead yeah 🚀